<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="input/rois0" default="/perception/object_recognition/detection/rois0"/>
  <arg name="camera/rois0/camera_info" default="/sensing/camera/camera0/camera_info"/>
  <arg name="output/rois0/objects" default="/perception/object_recognition/detection/roi_based_detector/rois0/objects"/>
  <arg name="input/rois1" default="/perception/object_recognition/detection/rois1"/>
  <arg name="camera/rois1/camera_info" default="/sensing/camera/camera1/camera_info"/>
  <arg name="output/rois1/objects" default="/perception/object_recognition/detection/roi_based_detector/rois1/objects"/>
  <arg name="param_path" default="$(find-pkg-share roi_based_detector)/config/roi_based_detector.param.yaml"/>

  <node pkg="roi_based_detector" exec="roi_based_detector_node" name="roi_based_detector_node" output="screen">
    <remap from="input/rois0" to="$(var input/rois0)"/>
    <remap from="input/rois0/camera_info" to="$(var camera/rois0/camera_info)"/>
    <param name="output/rois0/objects" value="$(var output/rois0/objects)"/>
    <remap from="input/rois1" to="$(var input/rois1)"/>
    <remap from="input/rois1/camera_info" to="$(var camera/rois1/camera_info)"/>
    <param name="output/rois1/objects" value="$(var output/rois1/objects)"/>
    <param from="$(var param_path)"/>
  </node>
</launch>
